android/app/src/main/java/org/openbot/common/Utils.java

```diff
@@ -22,7 +22,7 @@ public static JSONObject getStatus(
       boolean loggingEnabled,
       boolean noiseEnabled,
       boolean networkEnabled,
-      int driveMode,
+      String driveMode,
```

Comment:
Why does this need to be a string? This change breaks other parts of the app, where the driveMode.getValue() is passed.

Reply:
This can easily be fixed by replacing it with driveMode.toString(). But the question is what the rationale behind this change is.

Reply:
The reason it is a string is that we would not want to controller to interpret these numeric values. For example, if we pass 1, this has meaning in the bot app, but the controller does not need to know that 1 means "DUAL-DRIVE" for example. The controller would ideally just display what it is sent. This decouples them, in case these values change in the future, or add more options.

Reply:
Ok, makes sense. I have already updated this in the new UI fragment.